cmd/main.go

```diff
@@ -331,6 +377,16 @@ func scanEvoChain(ctx context.Context, c *config.Config, client scan.EthClient,
 				break
 			}
 
+			headers, err := client.HeaderByNumber(ctx, big.NewInt(int64(nextStartingBlock)))
+			if err != nil {
+				slog.Error("error retrieving block headers", "err", err.Error())
+				break
+			}
+
+			if err = tx.SetCurrentEvoBlockTimestamp(headers.Time); err != nil {
+				slog.Error("error storing block headers", "err", err.Error())
+				break
+			}
```

Comment:
I believe the code should be placed before the line `nextStartingBlock := lastScannedBlock.Uint64() + 1`, and we should store `lastScannedBlock` instead. However, I don't think this will impact the behavior.